pkg/render/intrusion_detection.go

```diff
@@ -1404,18 +1409,25 @@ func (c *intrusionDetectionComponent) adPersistentVolumeClaim() *corev1.Persiste
 
 func (c *intrusionDetectionComponent) adAPIDeployment(configureADStorage bool) *appsv1.Deployment {
 	var adModelVolumeSource corev1.VolumeSource
+	sc := securitycontext.NewBaseContext(securitycontext.RunAsUserID, securitycontext.RunAsGroupID)
+
 	if configureADStorage {
 		adStorageClassName := c.cfg.IntrusionDetection.Spec.AnomalyDetection.StorageClassName
 		adModelVolumeSource = corev1.VolumeSource{
 			PersistentVolumeClaim: &corev1.PersistentVolumeClaimVolumeSource{
 				ClaimName: adStorageClassName,
 			},
 		}
+
+		// When Persistent Storage is configured for AD API, it will use the host path of /var/anomaly-detection-api
```

Comment:
Isn't it /mnt?

Reply:
updated comments https://github.com/tigera/operator/pull/2300/commits/2932b4024aeb3e547363a5f0421635d82eb7ebae